src/adapters/handlers/nfts.ts

```diff
@@ -63,6 +63,9 @@ export function createNFTsHandler(
     const rentalStatus = params
       .getList<RentalStatus>('rentalStatus', RentalStatus)
       .concat(params.getList<RentalStatus>('rentalStatus[]', RentalStatus)) // concats to support both ways of sending the array
+    const creator = params
+      .getList('creator')
+      .concat(params.getList('creator[]')) // concats to support both ways of sending the array
```

Comment:
if we want to introduce this new way of supporting array, I would add it instead to the `getList` helper so it can be used for all array params.

Reply:
good idea! added the the `getList` helper :)